.github/workflows/firebase-merge.yml

```diff
@@ -13,16 +13,15 @@ jobs:
       - uses: actions/checkout@v2
       - run: npm install -g yarn
       - run: yarn
+      - run: cd src
+      - run: npm install firebase-functions@latest firebase-admin@latest --save
+      - run: npm install -g firebase-tools
+      - run: firebase --debug --only functions deploy 
```

Comment:
Deployer needs to authenticate himself before calling `deploy` command. Locally I am using `firebase deploy`. Not sure how to do that in actions because after the command is executed, user interaction is required